src/uu/split/src/split.rs

```diff
@@ -52,14 +52,86 @@ const AFTER_HELP: &str = help_section!("after help", "split.md");
 
 #[uucore::main]
 pub fn uumain(args: impl uucore::Args) -> UResult<()> {
+    let args = args.collect_lossy();
```

Comment:
I'd like to avoid this where possible. I think we should be operating on `OsStr` if we can.

Reply:
Unfortunately I had to keep the same implementation as in `fold` , i.e. using `uucore::Args`s `collect_lossy()` method. There is also `collect_ignore()` defined, but I think  to stay consistent we should use the same approach as `fold`

Reply:
ok, took another pass at it and in the latest commit removed the need for `collect_lossy()` and let `clap` panic on invalid non UTF-8 arguments + tests to cover that